en/config/safety.md

```diff
@@ -79,14 +80,14 @@ Additional (and underlying) parameter settings are shown below.
 
 Parameter | Setting | Description
 --- | --- | ---
-<a id="COM_RC_LOSS_T"></a>[COM_RC_LOSS_T](../advanced_config/parameter_reference.md#COM_RC_LOSS_T) | RC Loss Timeout | Time after RC stops updating supplied data that the RC link is considered lost. This must be kept short because the vehicle will continue to fly using the old RC data until the timeout triggers.
+<a id="COM_RC_LOSS_T"></a>[COM_RC_LOSS_T](../advanced_config/parameter_reference.md#COM_RC_LOSS_T) | Manual Control Loss Timeout | Time after RC stops updating supplied data that the RC link is considered lost. This must be kept short because the vehicle will continue to fly using the old RC data until the timeout triggers.
```

Comment:
I updated a bit so I could ask this question - is this correct? I mean is this correct and sufficiently generic for both Joystick and RC.

```suggestion
<a id="COM_RC_LOSS_T"></a>[COM_RC_LOSS_T](../advanced_config/parameter_reference.md#COM_RC_LOSS_T) | Manual Control Loss Timeout | Time after last valid data that the link is considered lost. This must be kept short because the vehicle will continue to fly using the last trusted data until the timeout triggers.
```

Reply:
I've proposed something else in 2e47be2ddea2dc5b6d7fa63a822af71bf966fcfd

Reply:
Excellent.